apps/web/package.json

```diff
@@ -3,9 +3,10 @@
   "version": "1.0.0",
   "private": true,
   "scripts": {
-    "dev": "next dev",
-    "build": "next build",
-    "start": "next start",
+    "env:load": "dotenv -e ../../.env --",
```

Comment:
We need to load the env here, any suggestions on how to do it in the root repo?

Reply:
If we move this command `env:load`, to root `package.json`, and run it on turbo repo pipeline, I believe it can work, but I never try it, what do you think?

I'm not had a huge experience using turbo repo pipelines, but it makes sense for me, if it is not possible please let me know.

And once again, thank you, you brought a lot of knowledge and good approaches with this PR.

Reply:
If you believe is better try it on onther PR, let me know, we can continue with it, dont worry.

Reply:
I also don't have much experience with turborepo, I think we can move on and try in another PR